backend/compact-connect/lambdas/cloudfront-csp/index.js

```diff
@@ -1,5 +1,337 @@
-// Placeholder for future UI Content Security Policy settings
+//
+//  index.js
+//  CompactConnect
+//
+//  Created by InspiringApps on 7/22/2024.
+//
+
+// ============================================================================
+//                               CONFIGURATION                                =
+// ============================================================================
+/**
+ * Configuration of supported domains per environment.
+ * @type {object}
+ */
+const environments = {
+    csg: {
+        prod: {
+            webFrontend: `app.compactconnect.org`,
+            dataApi: `api.compactconnect.org`,
+            s3Upload: ``, // @TODO: Waiting for environment configuration.
+        },
+        test: {
+            webFrontend: `app.test.compactconnect.org`,
+            dataApi: `api.test.compactconnect.org`,
+            s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-ivslg22f2yfz.s3.amazonaws.com`,
+        },
+    },
+    ia: {
+        prod: {
+            webFrontend: `app.jcc.iaapi.io`,
+            dataApi: `api.jcc.iaapi.io`,
+            s3Upload: ``, // @TODO: Waiting for environment configuration.
+        },
+        test: {
+            webFrontend: `app.test.jcc.iaapi.io`,
+            dataApi: `api.test.jcc.iaapi.io`,
+            s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-4h1anohxetmp.s3.amazonaws.com`,
+        },
+        justin: {
+            webFrontend: `app.justin.jcc.iaapi.io`,
+            dataApi: `api.test.jcc.iaapi.io`,
+            s3Upload: `test-persistentstack-mockbulkuploadsbucket0e8f27eb-4h1anohxetmp.s3.amazonaws.com`,
+        },
+    },
+};
+
+// ============================================================================
+//                                   HELPERS                                  =
+// ============================================================================
+/**
+ * Get the request domain from the lambda event record.
+ * @param  {object} eventRecord The cloudfront record from the lambda event.
+ * @return {string}             The bare domain of the request domain.
+ */
+const getRequestDomain = (eventRecord) => {
```

Comment:
Is it your style guide's standard to use arrow functions here? (What style standards, if any, are you using for JS?)

Reply:
Yes, we're favoring arrow functions over the `function` keyword (& its associated differences). Our linter styles are using the AirBnB rules as a base, with a small number of modifications.

Reply:
Got it!